src/main/webapp/src/app/recommendation/recommendation-item/recommendation-item.component.html

```diff
@@ -3,12 +3,13 @@
     <mat-card-header>
       <div mat-card-avatar class="example-header-image"></div>
       <mat-card-title id="name">{{hiddenGem.name}}</mat-card-title>
-      <mat-card-subtitle id="address">{{hiddenGem.vicinity}}</mat-card-subtitle>
+      <mat-card-subtitle id="address">{{hiddenGem.address}}</mat-card-subtitle>
     </mat-card-header>
     <img mat-card-image height="200" src={{photoUrl}} alt={{hiddenGem.name}}>
     <mat-card-content>
       <!-- TODO: Price level to be added after Places Details Request -->
-      <!-- <p id="price">Price Level (💲): {{hiddenGem.price_level}}</p> -->
+      <p id="price">Price Level (💲): {{hiddenGem.priceLevel}}</p>
+      <p id="bbbbeee">Score: {{hiddenGem.matchScore}}</p>
```

Comment:
maybe update the id? 
Also, instead of calling it "score:" maybe have something clearer like "This hidden gem satisfy {{hiddenGem.matchScore}} preference(s) out of the 3 you have set"

Reply:
yep changed it. thanks for finding this. i added in to help debug. but this is a great idea. updated now